app/src/screens/Home/index.tsx

```diff
@@ -1,27 +1,77 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
-
-const Home: React.FC = () => {
-  return (
-    <View style={styles.container}>
-      <Text style={styles.title}>Ahoy!</Text>
-    </View>
-  );
-};
+import { useAuth0 } from 'react-native-auth0';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Image,
+  TouchableHighlight,
+} from 'react-native';
 
 const styles = StyleSheet.create({
   container: {
+    backgroundColor: '#F4F4F6',
+    height: '100%',
     display: 'flex',
-    flexDirection: 'column',
+    alignContent: 'center',
+  },
+  body: {
+    marginHorizontal: 40,
+    marginTop: 30,
+  },
+  client: {
+    marginTop: 40,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
     alignItems: 'center',
-    height: '100%',
   },
-  title: {
-    fontSize: 48,
-    flexGrow: 1,
-    textAlignVertical: 'center',
-    color: 'black',
+  userText: {
+    fontWeight: '700',
+    fontSize: 24,
+    marginTop: 8,
+  },
+  logoContainer: {
+    marginTop: 100,
+    marginLeft: 18,
+  },
+  logoutText: {
+    color: '#0263E0',
+    textDecorationLine: 'underline',
+    padding: 0,
   },
 });
 
+const Home: React.FC = () => {
+  const { user, clearSession } = useAuth0();
+
+  const onLogout = async () => {
+    try {
+      await clearSession();
+    } catch (e) {
+      console.log(e);
+    }
+  };
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.logoContainer}>
+        <Image
+          source={require('../../../assets/icons/logo-twilio-red.png')}
```

Comment:
Nitpick: let's separate this as an "import". See the other components for an example.